assets/cluster-monitoring-operator/prometheus-rule.yaml

```diff
@@ -87,6 +89,14 @@ spec:
               kube_node_status_capacity{resource="cpu",unit="core"}
             )
           )
+          or on(node)
+          (
+            cluster:infra_nodes
+            * on(node) group_left() max by(node)
+            (
+              kube_node_status_capacity{resource="cpu",unit="core"}
+            )
+          )
```

Comment:
I was going to ask whether we really want to include infra nodes in `cluster:capacity_cpu_cores:sum`, as I would interpret that as user available capacity. But I see master nodes are already counted toward this, so I suppose infra nodes should be counted as well.

Reply:
I think the 'user available capacity' is typically seen in the \*workload\* metrics.  Though Ryan could correct me if I'm wrong.